### DISCUSSION TITLE
Path formatting with albumtypes

### DISCUSSION THREAD

**Discussion by simonmcnair** [beetbox/beets#4752] [Q&A] [answered]
Hi,
I'd like a directory format as follows:

Various Artists/OST/The Matrix
Various Artists/Compilation/Now 16

To be honest I'd like it if it could do:
Various Artists/GameThemes/Portal2/

Is any of these possible please ?

**Comment by sampsyo** [comment, marked as answer]
Hi! Have you tried configurations along the lines of the `albumtype:soundtrack` example in the docs?
https://beets.readthedocs.io/en/stable/reference/config.html#path-format-configuration

**Reply by simonmcnair** [reply]
> Hi! Have you tried configurations along the lines of the `albumtype:soundtrack` example in the docs? https://beets.readthedocs.io/en/stable/reference/config.html#path-format-configuration

Ahhh, that's a mybad.  I was looking in the wrong place.  https://beets.readthedocs.io/en/stable/reference/pathformat.html
